ui/src/reducers/dialogReducer.js

```diff
@@ -40,6 +42,54 @@ export const addToPlaylistDialogReducer = (
   }
 }
 
+export const DOWNLOAD_MENU_ALBUM = 'album'
+export const DOWNLOAD_MENU_ARTIST = 'artist'
+export const DOWNLOAD_MENU_PLAY = 'playlist'
+export const DOWNLOAD_MENU_SONG = 'song'
```

Comment:
I think these constants should be in the `actions/dialogs.js` file, as they are used only (AFAICT) as parameter to `openDownloadMenu` action